src/assets/drizzle/scripts/drizzle.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+import 'prismjs';
+
+const dom = {};
+
+dom.nav = document.getElementById('nav');
+dom.navMenu = document.getElementById('nav-menu');
+dom.navToggle = dom.nav.querySelector('a[href="#nav"]');
+dom.navLinks = dom.navMenu.querySelectorAll('a');
+
+function setActiveNavItem (pathname) {
```

Comment:
I don't quite understand...

I like the idea of using a Handlebars helper (feel free to make an issue for that). Regarding the page vs. hash, I think it already does acknowledge only the page and not the hash (though I definitely haven't tested it thoroughly).